docs/gpu.md

```diff
@@ -0,0 +1,61 @@
+# Using GPUs inside containers
+
+nerdctl provides docker-compatible NVIDIA GPU support.
+
+## Prerequisites
+
+- NVIDIA Drivers
+  - Same requirement as when you use GPUs on Docker. For details, please refer to [the doc by NVIDIA](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#pre-requisites).
+- `nvidia-container-cli`
+  - containerd relies on this CLI for setting up GPUs inside container. You can install this via [`libnvidia-container` package](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/arch-overview.html#libnvidia-container).
+
+## Options for `nerdctl run --gpus`
+
+`nerdctl run --gpus` is compatible to [`docker run --gpus`](https://docs.docker.com/engine/reference/commandline/run/#access-an-nvidia-gpu).
+
+You can specify number of GPUs to use via `--gpus` option.
+The following example exposes all available GPUs.
+
+```
+nerdctl run -it --rm --gpus all ubuntu:20.04 nvidia-smi
```

Comment:
`ubuntu:20.04` -> `nvidia/cuda:9.0-base ` might be more useful?

Reply:
Fixed this.